### DISCUSSION TITLE
vips_icc_export() using ICCs with D50 illuminance

### DISCUSSION THREAD

**Discussion by sjards** [libvips/libvips#3965] [Q&A] [answered]
Hello,
I'd like to use the ICC import and export functions within a current workflow I have for preparing images for printing.
But noticed that the ICC import and export specifically state the connection space as CIELAB D65.
Most printing ICCs are calculated for D50 illuminate.

I was wondering what impact this would have, and If noticeable, could I compensate with a D65 to D50 conversion after vips_export()?

Many thanks for any advise

Regards
Steve

**Comment by jcupitt** [comment, marked as answer]
Hi @sjards,

libvips should do this for you automatically. It always uses D65 for PCS, but will convert to D50 if necessary during export to device space. 

It uses Bradford for D65 -> D50, which I think is also what photoshop uses.

**Comment by sjards** [comment]
Hi @jcupitt,

Many thanks for the quick reply, and all the excellent work with libvips! 👍 
Brilliant to read libvips with convert automatically, I've not yet been able to compare results against software like Photoshop, but hope to do so soon.

All the best
Steve
